proto/speedtile.proto

```diff
@@ -40,6 +39,12 @@ message SpeedTile {
     repeated uint32 nextSegmentDelayVariances = 19 [packed=true];       //variance of delay samples from corresponding entry
     repeated uint32 nextSegmentQueueLengths = 20 [packed=true];         //length of any queue on segment from corresponding entry
     repeated uint32 nextSegmentQueueLengthVariances = 21 [packed=true]; //variance of queue length samples from corresponding entry
+
+    repeated uint32 referenceSpeed20 = 22 [packed=true]; //20% are this or slower than this specific ref speed for each segment
```

Comment:
since these are arrays maybe rename to:
referenceSpeeds20
referenceSpeeds40
referenceSpeeds60
referenceSpeeds80

Reply:
I've seen arguments both ways for this (pluarl names and non-plural). But since other repeated members have plural names I think these should also be plural.

Reply:
ok I've made the changes.  I have a few updates to check in shortly